content/en/database_monitoring/guide/apm-linking.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Linking Database Monitoring with APM
+kind: guide
+beta: true
+private: true
+---
+
+<div class="alert alert-warning">
+The features discussed on this page are in private beta. Contact your Customer Success Manager to learn more about it.
+</div>
+
+This guide assumes that you have configured [Datadog Monitoring](/database_monitoring/#getting-started) and are using [APM](/tracing/).
+
+## Before you begin
+
+Supported tracers
+: [dd-trace-go](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1) >= 1.42.0 (support for [database/sql](https://pkg.go.dev/database/sql) and [sqlx](https://pkg.go.dev/github.com/jmoiron/sqlx) packages)
+
+Supported databases
+: postgres, mysql
+
+Supported Agent versions
+: 7.36.1+
```

Comment:
for supported agent versions should we just do the most recent one?

Reply:
I just copied what we had in other pages to be consistent.

Reply:
that makes sense! i think we should update that so people always install the latest version 

I can do it in a separate PR though

Reply:
Yes, let's do that. 🙇